docs/src/installation.md

```diff
@@ -79,7 +79,7 @@ set_optimizer_attributes(model, "LogLevel" => 1, "PrimalTolerance" => 1e-7)
     repository for more details on how to use a particular solver, including any
     solver-specific options.
 
-### Supported solvers
+## Supported solvers (and applicability for problems)
```

Comment:
```suggestion
## Supported solvers
```
So we can make this a level-2 header, but changing the title name will break all links to the current header.